Microsoft.Toolkit.Uwp.UI.Controls/InAppNotification/InAppNotification.cs

```diff
@@ -132,9 +180,32 @@ public void Show(UIElement element, int duration = 0)
         /// <param name="duration">Displayed duration of the notification in ms (less or equal 0 means infinite duration)</param>
         public void Show(DataTemplate dataTemplate, int duration = 0)
         {
-            ContentTemplate = dataTemplate;
-            Content = null;
-            Show(duration);
+            bool shouldDisplayImmediately = ShouldDisplayImmediately();
```

Comment:
Could we use Generics or another pattern here to simplify these three different types of template objects?  Then the show logic could be centralized rather than triplicated, eh?

Reply:
I can surely create a function for it.

Reply:
Done.